package.json

```diff
@@ -42,11 +42,11 @@
   },
   "homepage": "https://github.com/oceanprotocol/ocean.js#readme",
   "peerDependencies": {
-    "web3": ">=1.3.5"
+    "web3": "^1.6.0"
```

Comment:
Something went wrong on this PR with the code comments. So this change for me was shown within our `dependencies` Where I then said it’s too adventurous. For the `peerDependencies` This range should stay as is though, so we do not force everybody to upgrade their web3.js, that’s why we used `>=` for this which is fine, but only for `peerDependencies`